ivadomed/metrics.py

```diff
@@ -177,7 +177,6 @@ def multi_class_dice_score(im1, im2):
     n_classes = im1.shape[0]
 
     for i in range(n_classes):
-        dice_per_class += dice_score(im1[i,], im2[i,]) \
-            if not dice_score(im1[i,], im2[i,]) == np.nan else 1.0
+        dice_per_class += dice_score(im1[i,], im2[i,]) if dice_score(im1[i,], im2[i,]) is not np.nan else 1.0
```

Comment:
What about using a parameter in this function `empty_score=1.0` as done in the `dice_score` --> so that it is transparent what we are doing with empty images?